### ISSUE TITLE
Migrating DMPOnline_v4 through MySQL aborts at ChangeLongStringsToText migration

### ISSUE THREAD

**Issue by jollopre** [DMPRoadmap/roadmap#1315]
An user from the community has attempted to migrate from DMPOnline_v4 to our new roadmap version, however the command:
```
rake db:migrate
```
has failed at the migration version = 20180123161959 (e.g. ChangeLongStringsToText). The reason for the failing is due to having defined defaults for strings at:
https://github.com/DMPRoadmap/roadmap/blob/master/db/migrate/20171024214257_add_links_to_orgs.rb#L3 and https://github.com/DMPRoadmap/roadmap/blob/master/db/migrate/20171124133802_add_links_to_templates.rb#L3

while attempting to preserve the defaults when the columns are changed to text, which is not supported by msql.

**Comment by jollopre** [comment]
The following steps have been applied to successfully migrate the user to our new roadmap db status:

1. Commented out line https://github.com/DMPRoadmap/roadmap/blob/master/db/migrate/20171102164156_change_plan_visibility_default.rb#L3
2. Executed:
```
rake db:migrate VERSION=20171013152425
```
3. Remove default from https://github.com/DMPRoadmap/roadmap/blob/master/db/migrate/20171024214257_add_links_to_orgs.rb#L3
4. Remove default from https://github.com/DMPRoadmap/roadmap/blob/master/db/migrate/20171124133802_add_links_to_templates.rb#L3
5.
```
rake db:migrate
```
We need to assess if is best to modify these specific migrations or trying to find a more optimal solution.

**Comment by stephaniesimms** [comment]
hope to incorporate changes into next release v1.0.1

**Comment by briri** [comment]
I removed the defaults from those migration files. Another option would have been to add a step that removes the defaults before converting the fields to TEXT. Both approaches update historic migrations and result in the same thing, so I think this approach is fine.

I pulled down v0.3.8 and did a `db:schema:load` to set the DB to a state that the user was working from. Then checked out development and ran `db:migrate` successfully.
